scss/lessonsChallenges.scss

```diff
@@ -0,0 +1,47 @@
+.profile-submissions{
+
+  .lesson_challenges{
+    display: flex;
+    margin-top: 15px
+  }
+
+  .lessons_challenges_card_body{
+    margin-left: 30px;
+  }
+
+  .display_lessons{
+    margin-top: 40px;
+  }
+  .lesson_title_container{
+    display: block;
+  }
+
+  .challenge_status{
+    height: 7px;
+    width: 40px;
+    padding: 10px;
+    margin-left: 2px;
```

Comment:
`margin-right` is better. Otherwise your block will have a 2px offset from the words